src/FieldWarning/Assets/Camera/SlidingCameraBehaviour.cs

```diff
@@ -354,6 +354,12 @@ private void TiltCameraIfNearGround(float oldAltitude)
 
     private void ClampCameraAltitude()
     {
+        if (_session.TerrainMap == null)
+        {
+            Debug.Log("Unable to clamp camera altitude. No terrain data available.");
+            return;
+        }
```

Comment:
Same as the minimap remark, needs a comment, and probably should not log anything.